lib/routes/api/forgot-password.js

```diff
@@ -6,6 +6,7 @@ const {validateEmail, emailSimpleText} = require('../../utils/email-utils');
 const {promisePool} = require('../../db');
 const {cacheClient} = require('../../helpers');
 const sysError = require('../error');
+const { assert } = require('@jambonz/lamejs/src/js/common');
```

Comment:
why are we requiring assert from this library, versus simply `const assert = require('assert')`?

Reply:
to be fixed in upcoming PR